### DISCUSSION TITLE
statues busy

### DISCUSSION THREAD

**Discussion by tomlave** [opengisch/QField#2519] [Q&A]
hello everyone,
I have a small question about the busy statue,
what are the other statues and what are their different functions ?
![Busy](https://user-images.githubusercontent.com/99726636/154069237-5e4cdd0d-3f41-4416-8ce5-90426b0b6adc.png)
?

**Comment by suricactus** [comment]
Hey, @tomlave. Nice spot!

The status is a new functionality coming to QFieldCloud next week. It will let you know if you project is ready to be used on your QField device. 

Currently you are seing all your projects showing "busy", until we update the QFieldCloud server. Please be patient for a few more days when you see your projects either green or red.
